src/article_metrics/api_v2_logic.py

```diff
@@ -167,3 +169,31 @@ def summary(page, per_page, order):
     start_pos = per_page * (page - 1) # slices are 0-based
     offset = start_pos + per_page
     return len(rows), rows[start_pos:offset]
+
+
+@cache(use=TWO_MIN_CACHE)
```

Comment:
Not sure if the cache could be pushed to the infrastructure level?

Reply:
It is done like this elsewhere in the service when some 3rd party calls are made so I followed suit. Plus, this is caching the 3rd party call (Crossref), not the actual endpoint response.